test/sequential/test-timers-block-eventloop.js

```diff
@@ -2,21 +2,23 @@
 
 const common = require('../common');
 const fs = require('fs');
+const commonTimeout = common.platformTimeout;
 
 const t1 = setInterval(() => {
-  common.busyLoop(12);
-}, 10);
+  common.busyLoop(commonTimeout(12));
```

Comment:
I'm not sure we should use `common.platformTimeout()` here. I think we don't want to increase the duration of the "busy" loop.